server/src/international_application_processor/onboarding_messages.go

```diff
@@ -0,0 +1,327 @@
+package international_application_processor
+
+import (
+	"fmt"
+	"html"
+	"strings"
+
+	"github.com/dxe/adb/mailer"
+	"github.com/dxe/adb/model"
+)
+
+const almiraTannerSignatureHtml = `
+		<p>
+			<b>Almira Tanner</b><br/>
+			Lead Organizer<br/>
+			Direct Action Everywhere<br/>
+			she/her
+		</p>
+	`
+
+func (b *onboardingEmailMessageBuilder) nearSFBayChapter() (mailer.Message, error) {
+	var msg mailer.Message
+	msg.FromName = sfBayCoordinator.Name
+	msg.FromAddress = sfBayCoordinator.Address
+	msg.ToName = b.fullName
+	msg.ToAddress = b.email
+	msg.Subject = "Join your local Direct Action Everywhere chapter!"
+
+	var body strings.Builder
+
+	fmt.Fprintf(&body, "<p>Hey %v!</p>", html.EscapeString(b.firstName))
+
+	body.WriteString(
+		`<p>
+			Thank you for reaching out about getting involved with Direct Action
+			Everywhere.
+		</p>
+		<p>
+			Great news—the SF Bay Area chapter is within 100 miles of you.
+			We’d love to see you at an upcoming event!
```

Comment:
nit: this sounds kinda robotic ("within 100 miles of you")... should we say something a bit more conversational like "your near the SF Bay chapter" or something?